src/dashboard/src/media/js/misc.js

```diff
@@ -139,3 +139,23 @@ $(document).ready(
                 $(this).closest('.preview-help-text').children('.preview').show();
               });
 });
+
+// Return true if user navigated to page via browser back/forward button.
+// Return false if user navigated directly or if browser doesn't support Navigation Timing API.
+function navigatedFromBackOrForwardButton() {
+  if (window.performance) {
```

Comment:
This is a small thing, but if we align these checks to the happy path, and flip the IF to return earlier we can bring the indents in by one: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 

So:
```javascript
  if !(window.performance) {
    return false;
  }
  // Navigation Timing v2 spec
  var navEntries = window.performance.getEntriesByType('navigation');
  if (navEntries.length > 0 && navEntries[0].type === 'back_forward') {
    return true;
  // Navigation Timing v1 spec (deprecated but widely supported)
  } else if (window.performance.navigation
      && window.performance.navigation.type == window.performance.navigation.TYPE_BACK_FORWARD) {
    return true;
  } else {
    return false;
  }
```
It reduces visual complexity a little which I feel is important here where we end the function with:
```javascript
      return true;
    } else {
      return false;
    }
  } else {
    return false;
  }
}
```
It gets harder to switch back between the lines. 

I think the other if/else if/else might be fine I didn't think too hard about it! :slightly_smiling_face: